### DISCUSSION TITLE
Building complex Layout with MJML like in the example provided?

### DISCUSSION THREAD

**Discussion by Wellbios** [mjmlio/mjml#2803] [Q&A]
Hi all, 

even though Im not an expert (but also not a newbie when it comes to Html/CSS Email anymore), I would like to know if it is possible to build something like this in MJML? [https://viewstripo.email/template/7b57654a-d8f9-4351-9166-aeb7c5d6000d](url)

I've tried to replicate it but it gave me not the desired result. So wanted to give this a try without the hassle to optimize it for all the email clients.

Also do you use MJML in your day to day job (as an Email Dev for example. Just curious to know)

Thanks in advance :).

**Comment by iRyusa** [comment]
Buttons on some elements might need to be done manually instead of using mj-button but there's no blocking thing in this layout
